packages/api/src/utils/github/buildUserQuery.ts

```diff
@@ -0,0 +1,7 @@
+import { User } from '../../entities/User';
+
+export const buildUsersQuery = (users: User[]) => {
+  const usersQuery = users.map((user) => `author:${user.githubUsername}`).join(' ');
+
+  return usersQuery;
+};
```

Comment:
Unless you think this is going to be expanded rather significantly, I'd just declare this where it's being used